### PR TITLE
update graphql-apollo-react-update

### PR DESCRIPTION
**Pull request by indiepopart**
Updates [Easily Consume a GraphQL API from React with Apollo](https://developer.okta.com/blog/2021/01/08/graphql-apollo-react) to streamline setup.

### CODE REVIEW

File: _source/_posts/2021-01-08-graphql-apollo-react.md [hunk 1]
```diff
@@ -28,19 +28,16 @@ If you prefer to watch the demo, you can view this walk through.
...
 
-The first thing you will need a free developer account from Okta. if you don't have one you can [sign up for one here.](https://developer.okta.com/signup/). Once you have completed that navigate to your developer's console and click on **Applications** and then **Add Application**. Select **Single Page App** and click **Next**.   
+{% include setup/cli.md type="spa" framework="React" loginRedirectUri="http://localhost:3000/callback" %}
 
-Give your application a meaningful name. You will also need to change your URIs to *localhost:3000* as this is the default in React.
-
-{% img blog/graphql-apollo-react/okta-app-settings.png alt:"Okta App Settings" width:"800" %}{: .center-image }
-
-Click on **Done**. On the application management page, you will be presented with your *Client id*. Make note of this as you will need it in your application.
+The OktaCLI will present your the *Client id*. Make note of this as you will need it in your application.
```
Comment (mraible) [inline _source/_posts/2021-01-08-graphql-apollo-react.md:33]:
```suggestion
The Okta CLI will present you with a *Client ID*. Make note of this as you will need it in your application.
```

File: _source/_posts/2021-01-08-graphql-apollo-react.md [hunk 2]
```diff
@@ -28,19 +28,16 @@ If you prefer to watch the demo, you can view this walk through.
 
 ## Create your Okta Application
 
-The first thing you will need a free developer account from Okta. if you don't have one you can [sign up for one here.](https://developer.okta.com/signup/). Once you have completed that navigate to your developer's console and click on **Applications** and then **Add Application**. Select **Single Page App** and click **Next**.   
+{% include setup/cli.md type="spa" framework="React" loginRedirectUri="http://localhost:3000/callback" %}
 
-Give your application a meaningful name. You will also need to change your URIs to *localhost:3000* as this is the default in React.
-
-{% img blog/graphql-apollo-react/okta-app-settings.png alt:"Okta App Settings" width:"800" %}{: .center-image }
```
Comment (mraible) [inline _source/_posts/2021-01-08-graphql-apollo-react.md:35]: Can you please delete this image too?

**Review by mraible — CHANGES_REQUESTED** [review]

File: _source/_posts/2021-01-08-graphql-apollo-react.md (hunk 2 above)
Comment (indiepopart) [reply _source/_posts/2021-01-08-graphql-apollo-react.md:35]: I see it deleted?

Comment (mraible) [reply _source/_posts/2021-01-08-graphql-apollo-react.md:35]: You deleted the line, but the file still exists in Git.

File: _source/_posts/2021-01-08-graphql-apollo-react.md [hunk 3]
```diff
@@ -28,19 +28,16 @@ If you prefer to watch the demo, you can view this walk through.
...
-
-Click on **Done**. On the application management page, you will be presented with your *Client id*. Make note of this as you will need it in your application.
+The Okta CLI will present your with a *Client ID*. Make note of this as you will need it in your application.
```
Comment (mraible) [inline _source/_posts/2021-01-08-graphql-apollo-react.md:33]:
```suggestion
The Okta CLI will present you with a *Client ID*. Make note of this as you will need it in your application.
```

**Review by mraible — APPROVED** [review]

**Pull request merged**